pygmt/__init__.py

```diff
@@ -19,7 +19,7 @@
 from .sampling import grdtrack
 from .mathops import makecpt
 from .modules import GMTDataArrayAccessor, config, info, grdinfo, which
-from .gridops import grdcut
+from .gridops import grdcut, gridfilter
```

Comment:
```suggestion
from .gridops import grdcut, grdfilter
```

Just a small typo, that's why it's not importing :smiley: